gulpfile.js

```diff
@@ -22,9 +22,9 @@ var fs = require("graceful-fs"),
     postcss = require('gulp-postcss'),
     rtl = require('postcss-rtl'),
     babel = require('gulp-babel');
-    // The startbootstrap theme gulpfiles do not yet support node 15 so need to be installed and built seperately.
-    // blogtheme = require('./src/OrchardCore.Themes/TheBlogTheme/wwwroot/gulpfile');
-    // comingsoontheme = require('./src/OrchardCore.Themes/TheComingSoonTheme/wwwroot/gulpfile');
+// The startbootstrap theme gulpfiles do not yet support node 15 so need to be installed and built seperately.
```

Comment:
Revert the changes on this file. We will asses the gulpfile issues in a different PR.

Reply:
Remove these commented lines and the comment line.